how-to/how-to-set-sandbox-config-kata.md

```diff
@@ -75,6 +75,8 @@ There are several kinds of Kata configurations and they are listed below.
 | `io.katacontainers.config.hypervisor.virtio_fs_cache` | string | the cache mode for virtio-fs, valid values are `always`, `auto` and `none` |
 | `io.katacontainers.config.hypervisor.virtio_fs_daemon` | string | virtio-fs `vhost-user` daemon path |
 | `io.katacontainers.config.hypervisor.virtio_fs_extra_args` | string | extra options passed to `virtiofs` daemon |
+| `io.katacontainers.config.hypervisor.rx_rate_limiter_max_rate` | uint64 | rx rate limiter for throttling network I/O inbound bandwidth |
```

Comment:
@GabyCT - this is failing Travis as "tx" and "rx" are not dictionary words. Please either change the wording, or [add these terms to the spell check dictionary](https://github.com/kata-containers/tests/tree/master/cmd/check-spelling#adding-a-new-word).

Reply:
Maybe this was meant to be directed at @Pennyzct?

Reply:
@jodh-intel , I think your review was for @Pennyzct